compiler/daml-extension/src/extension.ts

```diff
@@ -33,8 +35,10 @@ export async function activate(context: vscode.ExtensionContext) {
     // Get telemetry consent
     const consent = getTelemetryConsent(config, context);
 
-    // Check extension version to display release notes on updates
+    // Display release notes on updates
     showReleaseNotesIfNewVersion(context);
+    // Notify about new blog posts
+    showBlogIfNotSeen(config, context);
```

Comment:
What happens if this fails, e.g., due to a network error (might be caused by a firewall). I guess since this is `async` we will still continue? We definitely don’t want to prevent the rest of the extension from loading.

Reply:
Yes, I think it's safe because
1. It's async
2. I don't do anything in the error cases, e.g. if the `fetch` of the daml blog post fails, I just silently move on.

Reply:
Can we please verify that it is not harmful by running it with wifi turned off.